src/functionalTest/resources/scenarios/RIA-3690-remission-decision-rejected.json

```diff
@@ -0,0 +1,46 @@
+{
+  "description": "RIA-3690 Notification of remission decision rejected",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "AdminOfficer",
+    "input": {
+      "id": 3690,
+      "eventId": "recordRemissionDecision",
+      "state": "pendingPayment",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "remissionDecision": "rejected"
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "notificationsSent": [
+          {
+            "id": "3690_REMISSION_DECISION_PARTIALLY_APPROVED_LEGAL_REPRESENTATIVE",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          }
+        ]
+      }
+    },
+    "notifications": [
+      {
+        "reference": "3690_REMISSION_DECISION_REJECTED_LEGAL_REPRESENTATIVE",
+        "recipient": "{$TEST_LAW_FIRM_A_USERNAME}",
+        "subject": "\tImmigration and Asylum appeal: Remission application rejected",
+        "body": [
+          "PA/12345/2019",
+          "CASE001",
+          "Talha Awan",
+          "{$iaExUiFrontendUrl}"
+        ]
```

Comment:
Similar to above, once the GovNotify template has been updated, you will need to check for the following in the functional test:

"{$customerServices.telephoneNumber}",
"{$customerServices.emailAddress}"

Reply:
Added the telephoneNumber and emailAddress. Modified the template in Gov Notify